config/locales/en/activerecord/course/assessment/submission.yml

```diff
@@ -12,6 +12,7 @@ en:
         course/assessment/submission:
           experience_points_record:
             inconsistent_user: 'the creator of the submission must be the same as the course user'
+          submission_already_exists: "Submission with ID %{existing_id} already exists"
```

Comment:
I was thinking that they might ask for help. With the ID, tutors will be able to point them to the correct URL. If they try again, they'll probably click back, attempt, and end up with the same error.